web/src/locales/en/data.json

```diff
@@ -380,6 +381,7 @@
     "Is profile public - Tooltip": "After being closed, only global administrators or users in the same organization can access the user's profile page",
     "Modify rule": "Modify rule",
     "New Organization": "New Organization",
+    "Properties - Tooltip": "Properties of the organization",
```

Comment:
Modify translated words to `Properties of the object` to cover both meaning

Reply:
okay

Reply:
done